config/initializers/apipie.rb

```diff
@@ -0,0 +1,8 @@
+Apipie.configure do |config|
+  config.app_name                = "Cms"
+  config.api_base_url            = "/api"
+  config.doc_base_url            = "/apipie"
+  config.api_controllers_matcher = "#{Rails.root}/app/controllers/**/*.rb"
```

Comment:
Should this be pointing to the`ApiController`?

Reply:
no, because we want different controllers to be picked up